src/main/java/com/cen4010/bookstore/book/BookRepository.java

```diff
@@ -1,14 +1,24 @@
 package com.cen4010.bookstore.book;
 
+import java.util.Optional;
 import java.util.UUID;
 import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface BookRepository extends JpaRepository<Book, UUID> {
 
-  List<Book> findByGenre(String genre);
 
-  List<Book> findTop10ByOrderByCopiesSoldDesc();
+
```

Comment:
Why are you removing `findBy` methods others have explicitly added? Have you tested to make sure you're not breaking others code?

Reply:
GitHub diff stuff- again, my bad.